chainer/functions/pooling/unpooling_2d.py

```diff
@@ -0,0 +1,57 @@
+import numpy
+
+from chainer import cuda
+from chainer.functions.pooling import pooling_2d
+from chainer.utils import conv
+
+
+class Unpooling2D(pooling_2d.Pooling2D):
+
+    """Unpooling over a set of 2d planes."""
+
+    def __init__(self, ksize, stride=None, pad=0,
+                 outsize=None, cover_all=True, use_cudnn=True):
+        super(Unpooling2D, self).__init__(
+            ksize, stride, pad, cover_all, use_cudnn)
+        self.outh, self.outw = (None, None) if outsize is None else outsize
```

Comment:
As we did in `Deconvolution2D` [here](https://github.com/pfnet/chainer/blob/2042053a89b1be0536ce939c4971cb82b30f65ac/chainer/functions/connection/deconvolution_2d.py#L45), we should check outsize is appropriate by `get_conv_outsize`